tool/cmd/cmd-go.go

```diff
@@ -34,12 +42,12 @@ var commandGo = cli.Command{
 
 		err = setup.Setup(ctx)
 		if err != nil {
-			return cli.Exit(err, exitCodeFailure)
+			return ex.Errorf(err, "failed to build with toolexec with exitcode %d", exitCodeFailure)
```

Comment:
Side note: cli.exit was changed to ex.error to preserve the error chain. Otherwise, ex.fatal wouldn't be able to print the correct error stack.

Reply:
See below